templates/partials/candidates-office-table.html

```diff
@@ -0,0 +1,17 @@
+<table id="results" class="data-table" data-type="candidate">
```

Comment:
No strong preference, but if that change gets made, I think it can happen in a separate patch.